include/CL/sycl/handler.hpp

```diff
@@ -933,7 +933,9 @@ void set_args(Ts &&... args);
 #endif
       {
         __hipsycl_launch_kernel(detail::dispatch::device::parallel_for_kernel<KernelName>,
-                              grid, block, shared_mem_size, stream->get_stream(),
+                              detail::make_kernel_launch_range<dimensions>(grid),
```

Comment:
All the indentation in this change batches seems weird to me.

Reply:
you're right, thank you! I'll go over the indentation again and prettify things.